library/core/src/sync/atomic.rs

```diff
@@ -28,7 +28,8 @@
 //! Basically, creating a *shared reference* to one of the Rust atomic types corresponds to creating
 //! an `atomic_ref` in C++; the `atomic_ref` is destroyed when the lifetime of the shared reference
 //! ends. (A Rust atomic type that is exclusively owned or behind a mutable reference does *not*
-//! correspond to an "atomic object" in C++, since it can be accessed via non-atomic operations.)
+//! correspond to an “atomic object” in C++, since the underlying primitive can be mutably accessed
+//! with `get_mut to perform non-atomic operations.)
```

Comment:
```suggestion
//! with `get_mut` to perform non-atomic operations.)
```

... and maybe this should say "methods like `get_mut`" since there's also (unstable) `get_mut_slice`.

Reply:
Yes, that's better. Updated.